### ISSUE TITLE
Try changelog and auto versioning plugins?

### ISSUE THREAD

**Issue by cball** [echobind/eb-scripts#17]
Can we try out the following on this project to get a good feel for them?

```
@semantic-release/commit-analyzer
analyzeCommits: Determine the type of release by analyzing commits with conventional-changelog

@semantic-release/release-notes-generator
generateNotes: Generate release notes for the commits added since the last release with conventional-changelog
```

Seems like it would be a huge help if we can automate these steps on projects.

**Comment by jsjoeio** [comment]
Let's do it!

**Comment by jsjoeio** [comment]
Closed via #30
